### PR TITLE
Up 7

### PR DESCRIPTION
**Pull request by Ryan-Guinn**
This pull request adds the inventory page and fixes some project page issues.

The functionality has been extensively manually tested and has automated tests for the applicable react component.

This also adds initial testing set-up

### CODE REVIEW

File: packages/app/pages/AllProjectsPage/AllProjectsPage.tsx [hunk 1]
```diff
@@ -19,43 +21,140 @@ interface Props {
...
+   * Handler function for when buttons are pressed.
+   * TODO apply the filter to the user agents
+   * @param buttonName the text of the button that was clicked
+   */
+  handleClick = (buttonName) => {
+    this.setState({ activeButton: buttonName })
+    this.forceUpdate(() => console.log('Finished Updating'))
+    // this.state.activeButton  = buttonName
+    console.log("Clicked", buttonName)
+    // TODO refresh the list and filter
```
Comment (Ryan-Guinn) [inline packages/app/pages/AllProjectsPage/AllProjectsPage.tsx:52]: Try to add when TODOs need to be completed by.

For example
TODO before implementing material add
or something along those lines so someone implementing the affected functionality in the future can do a quick scan for todo's

File: packages/app/pages/AllProjectsPage/AllProjectsPage.tsx [hunk 2]
```diff
@@ -19,43 +21,140 @@ interface Props {
...
+
+  currentTheme = themeable(this.props.theme)
+  i = this.props.i
+
+  render() {
+    return (
+      <div {...this.currentTheme((this.i * 7) + 13, "medium-6",  "end", "columns")}>
+        <div {...this.currentTheme((this.i * 7) + 14, "projects_item")}>
+          <div {...this.currentTheme((this.i * 7) + 15, "item_row")}>
+            <span {...this.currentTheme((this.i * 7) + 16, "row_image")}><img src={this.props.org.image} /></span>
```
Comment (Ryan-Guinn) [inline packages/app/pages/AllProjectsPage/AllProjectsPage.tsx:122]: Avoid magic numbers and unnamed calculations

File: packages/ui-views/templates/SingleProjectTemplate/singleprojectTemplate.tsx [hunk 3]
```diff
@@ -1,34 +1,35 @@
-import * as React from 'react'
```
Comment (Ryan-Guinn) [inline packages/ui-views/templates/SingleProjectTemplate/singleprojectTemplate.tsx:1]: mass re-formatting should be on its own commit to make for easier hiding

File: packages/app/pages/AllProjectsPage/AllProjectsPage.tsx (hunk 2 above)
Comment (fontainesw) [reply packages/app/pages/AllProjectsPage/AllProjectsPage.tsx:122]: This should be better documented

**Review by ConnorHibbs — APPROVED** [review]

**Review by bensonjohn — APPROVED** [review]
